base/README.md

```diff
@@ -0,0 +1 @@
+docker build -t scanhub-base .
```

Comment:
Shouldn't this step be executed in docker-compose?

Reply:
I think this should be separated, because it builds the base not only for this composition.